### ISSUE TITLE
simple theme: search box

### ISSUE THREAD

**Issue by dalf** [searxng/searxng#347]
Currently:
![image](https://user-images.githubusercontent.com/1594191/134933560-c5174e9a-48d9-44ed-9e29-1ce5d9cb9ce3.png)

start.paulgo.io :
![image](https://user-images.githubusercontent.com/1594191/134933604-74efc7a7-a431-4fe9-9a21-d953592abe1f.png)

Google:
![image](https://user-images.githubusercontent.com/1594191/134933808-632c8a79-a798-48a5-8423-af71a646abbb.png)

Duckduckgo:
![image](https://user-images.githubusercontent.com/1594191/134933844-7b144891-cacc-45cd-b61b-19bb9d637aa8.png)

Brave:
![image](https://user-images.githubusercontent.com/1594191/134934414-342d03f0-a454-4a21-a499-a5b310cb9e08.png)

Qwant:
![image](https://user-images.githubusercontent.com/1594191/134934605-987fff0b-3afa-4c4f-8dad-6f6e538c09c6.png)

---

Suggestion (*):
![image](https://user-images.githubusercontent.com/1594191/134933751-f04d356e-b43f-415c-9bb4-88d4fc1a9284.png)

(*):
* to be slightly different from the Google layout, but it looks like the Duckduckgo layout...
* ... and I'm not a UI / UX designer, so it can the start.paulgo.io version too.
* It would be nice to have a preference to choose how compact the elements are (like line-height, it can be done later)

Why change? The current simple layout is really compact: it displays more data in the same space, but it is less readable. The usual layout of search engines have more space between the elements.

**Comment by mrpaulblack** [comment]
I really like your suggestions. The only things I would change:
* more padding on the top and bottom of the input field to make the search input field bigger
* a drop shadow on the search input field itself
* maybe add some icons for the different categories; we could use https://github.com/FortAwesome/Font-Awesome


* (the pref for compactness could change the line-height and the margin on the articles for example)

**Comment by dalf** [comment]
Quick and dirty changes: https://github.com/dalf/searxng/tree/simple-search-box

I will open a PR later, but it raises one question: there are two main colors : @color-base and @color-base-dark (see definitions.less).

With the suggested change, should all @color-base turn to be @color-base-dark ?

![image](https://user-images.githubusercontent.com/1594191/134955602-6d27281f-ee3b-4346-a9fa-9839848ae3b1.png)

![image](https://user-images.githubusercontent.com/1594191/134955636-554bcc97-5bd8-4e36-a01d-a69eac6f1c24.png)

**Comment by mrpaulblack** [comment]
Well my idea was to use CSS vars for the themes; so defenition.less would look something like that:

```css
.light {
  --color-base: white;
  --color-footer-border: dark-gray;
}

.dark {
  --color-base: black;
  --color-footer-border: light-gray;
}
```

(You would drop color-base-dark and color-base-light and just use one base color var instead; In the style.css all less vars would be needed to change to css vars: for example ```border: 1px solid var(--footer-border-color);```)

Browser support seems also good: https://developer.mozilla.org/en-US/docs/Web/CSS/var()#browser_compatibility

Then based on the theme the light or dark class can be applied to the body. We could even do a JS theme switcher based on device preference...

[EDIT]: Also I think the search logo on the left side of the search input field like every other search engine would be cool... What do you think?

**Comment by return42** [comment]
> maybe add some icons for the different categories; we could use https://github.com/FortAwesome/Font-Awesome

One feature of _categories_ is, that you can define your own categories e.g. when you configure ..

```diff
diff --git a/searx/settings.yml b/searx/settings.yml
index 3fefb62f..4dd7125a 100644
--- a/searx/settings.yml
+++ b/searx/settings.yml
@@ -600,10 +600,12 @@ engines:
     use_mobile_ui: false
     # additional_tests:
     #   android: *test_android
+    categories: google
 
   - name: google images
     engine: google_images
     shortcut: goi
+    categories: google
     # additional_tests:
     #   android: *test_android
     #   dali:
@@ -616,18 +618,21 @@ engines:
   - name: google news
     engine: google_news
     shortcut: gon
+    categories: google
     # additional_tests:
     #   android: *test_android
 
   - name: google videos
     engine: google_videos
     shortcut: gov
+    categories: google
     # additional_tests:
     #   android: *test_android
 
   - name: google scholar
     engine: google_scholar
     shortcut: gos
+    categories: google
 
   - name: google play apps
     engine: xpath
@@ -637,7 +642,7 @@ engines:
     url_xpath: './/div[@class="RZEgze"]//div[@class="kCSSQe"]//a/@href'
     content_xpath: './/div[@class="RZEgze"]//a[@class="mnKHRc"]'
     thumbnail_xpath: './/div[@class="uzcko"]/div/span[1]//img/@data-src'
-    categories: files
+    categories: google
     shortcut: gpa
     disabled: true
     about:
```
.. you will get a new category ...

![grafik](https://user-images.githubusercontent.com/554536/134960648-c43939f0-b31e-456a-8e43-c574729749c0.png)

This feature has only one drawback, you can't use (configure) translations for the new category name.

I see the same problem when we have icons for categories.

**Comment by mrpaulblack** [comment]
Well maybe we could add a macro (in the simple templates) with some if statements that applies a specific class (which is needed for a specific icon in font awesome) based on the category name; The custom categories could just get a default icon (if non of the if statements for the default categories match)...; This seems a little hacky tho...

**Comment by return42** [comment]
> Well maybe we could add ... This seems a little hacky tho...

Hm .. why not a macro (or any other solution) .. I pointed this out to have an eye on this feature .. at the end the UX counts, this was my intention.

**Comment by dalf** [comment]
Implemented in #355
